pom.xml

```diff
@@ -48,7 +48,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
-      <version>4.3.19.RELEASE</version>
+      <version>2.0.8</version>
```

Comment:
Thanks for changing this back. We can ignore the vulnerability warnings for now, and look for fix in the meantime. FYI @panerorenn9541